versioned_docs/version-3.x/developer/app-store/apps/avatax/configuration.mdx

```diff
@@ -132,6 +132,8 @@ When the AvaTax app is handling checkout or order calculate taxes webhooks it is
 
 For order confirmed webhook, the AvaTax app is using [price reduction discounts flow](https://developer.avalara.com/erp-integration-guide/sales-tax-badge/transactions/discounts-and-overrides/discounting-a-transaction/). It means that Saleor is responsible for calculating discounts and AvaTax app is sending discounted prices to Avalara for tax calculation.
 
+For more information about how discounts work in Saleor see discounts [guide](../../../../developer/discounts/overview) from our docs.
```

Comment:
```suggestion
For more information about how discounts work in Saleor see discounts [guide](developer/discounts/overview.mdx) from our docs.
```

Reply:
@aniav done, but that shouldn't be in the scope of this PR, as here the idea is to align canary and stable meaning they contain the same content, the requested change affects both so it's not an alignment :wink:

Reply:
Yup, that's why I said it's nonblocking 😂